install/securityapi/bin/sig-test.map

```diff
@@ -31,17 +31,17 @@
 # Note: Recording the signatures of a package includes all
 #       child packages.  The signature test tool looks for
 #       the best signature file to use when playing back
-#       signatures.  Meaning if we have a javax.servlet
-#       signature file and a javax.servlet.jsp signature file,
-#       the signature test tool will use the javax.servlet.jsp
-#       signature file to verify the javax.servlet.jsp package
-#       signatures even though the javax.servlet signature
-#       file contains the javax.servlet.jsp package signatures.
+#       signatures.  Meaning if we have a jakarta.servlet
+#       signature file and a jakarta.servlet.jsp signature file,
+#       the signature test tool will use the jakarta.servlet.jsp
+#       signature file to verify the jakarta.servlet.jsp package
+#       signatures even though the jakarta.servlet signature
+#       file contains the jakarta.servlet.jsp package signatures.
 #       The signatures are in both files (since the API Check
 #       tool records child package signatures and there does
 #       not seem to be a way to turn this feature off) but the
-#       javax.servlet.jsp signature file can vary independent
```

Comment:
can remove this file as install/securityapi/bin/sig-test_se8.map & install/securityapi/bin/sig-test_se11.map will be used.

Reply:
Deleted